custom/ilsgateway/tanzania/warehouse/updater.py

```diff
@@ -255,6 +259,7 @@ def populate_report_data(start_date, end_date, domain, runner, locations=None, s
         )()
         res.get()
     runner.location = None
+    runner.updating_historical_data = True
```

Comment:
if we're setting this to True on each run, it looks like it's going to not run update_historical_data the first time, but run it every subsequent time. is that intended?